### ISSUE TITLE
 Issue: Black Screen in Desktop and Browser Application using Plotly

### ISSUE THREAD

**Issue by AlexMayka** [flet-dev/flet#3015]
Dear Flet Developers,,

I am facing an issue while using the Flet library in combination with Plotly to create a chart in a Windows application. The problem is that when I run the application, the window opens, but only a black screen is displayed, and no content is visible. I have provided the code I am using below.

__Code example to reproduce the issue:__

```python
import plotly.express as px
import plotly.graph_objects as go
import flet as ft
from flet.plotly_chart import PlotlyChart

def main(page: ft.Page):
    page.bgcolor = ft.colors.WHITE
    labels = ["Oxygen", "Hydrogen", "Carbon_Dioxide", "Nitrogen"]
    values = [4500, 2500, 1053, 500]

    fig = go.Figure(data=[go.Pie(labels=labels, values=values)])

    page.add(ft.Row(controls=[ft.Text("alex", color=ft.colors.BLACK, size=25), PlotlyChart(fig, expand=True)]))
    page.update()

ft.app(target=main)
```
__________
The code above is supposed to create a simple chart using the Plotly library and display it on the Flet application window. However, when I run the code, the application window opens, but only a black screen is displayed. No content is visible, including the chart.

This issue occurs consistently every time I run the application. I have tried running the application in both the desktop and browser environments, but the same black screen is displayed in both cases. I have also tried running the application on different machines with different operating systems, but the issue persists.

__________

__Flet version (pip show flet):__
```commandline
Name: flet
Version: 0.22.0
Summary: Flet for Python - easily build interactive multi-platform apps in Python
Home-page:
Author: Appveyor Systems Inc.
Author-email: hello@flet.dev
License: Apache-2.0
Location: C:\Users\user\Desktop\Prog\Python\GUI\ProgOne\venv\Lib\site-packages
Requires: cookiecutter, fastapi, flet-runtime, packaging, qrcode, uvicorn, watchdog
Required-by:
```
_________

__Requirements.txt__
```commandline
annotated-types==0.6.0
anyio==4.3.0
arrow==1.3.0
binaryornot==0.4.4
certifi==2024.2.2
chardet==5.2.0
charset-normalizer==3.3.2
click==8.1.7
colorama==0.4.6
cookiecutter==2.6.0
fastapi==0.110.1
flet==0.22.0
flet-core==0.22.0
flet-runtime==0.22.0
h11==0.14.0
httpcore==1.0.5
httptools==0.6.1
httpx==0.27.0
idna==3.6
Jinja2==3.1.3
kaleido==0.2.1
markdown-it-py==3.0.0
MarkupSafe==2.1.5
mdurl==0.1.2
numpy==1.26.4
oauthlib==3.2.2
packaging==23.2
pandas==2.2.2
plotly==5.20.0
pydantic==2.6.4
pydantic_core==2.16.3
Pygments==2.17.2
pypng==0.20220715.0
python-dateutil==2.9.0.post0
python-dotenv==1.0.1
python-slugify==8.0.4
pytz==2024.1
PyYAML==6.0.1
qrcode==7.4.2
repath==0.9.0
requests==2.31.0
rich==13.7.1
six==1.16.0
sniffio==1.3.1
starlette==0.37.2
tenacity==8.2.3
text-unidecode==1.3
types-python-dateutil==2.9.0.20240316
typing_extensions==4.11.0
tzdata==2024.1
urllib3==2.2.1
uvicorn==0.29.0
watchdog==4.0.0
watchfiles==0.21.0
websockets==12.0
```

__________
I would greatly appreciate any help or insights you could provide to resolve this issue. Thank you for your time and assistance.

Best regards,
Alex

**Comment by AlexMayka** [comment]
The error persists anyway

```python
import plotly.express as px

import flet as ft
from flet.plotly_chart import PlotlyChart


def main(page: ft.Page):
    df = px.data.gapminder().query("continent=='Oceania'")
    fig = px.line(df, x="year", y="lifeExp", color="country")
    row = ft.Row(
        controls=[
            ft.Text("Alex"),
            PlotlyChart(fig, expand=True)
        ],
        expand=True
    )

    page.add(row)


ft.app(target=main)
```

**Comment by ndonkoHenri** [comment]
It shows up on my end (macOS): 

![image](https://github.com/flet-dev/flet/assets/98978078/11431751-340e-449b-ab8b-75fb74959432)

**Comment by AlexMayka** [comment]
It is possible that you might be using an additional setting or library that I don't have? Or perhaps it doesn't work on Windows 11?

**Comment by ndonkoHenri** [comment]
Ah, check your terminal. I got a warning telling me to install pandas, then another one to install kaleido.
```bash
Image export using the "kaleido" engine requires the kaleido package,
which can be installed using pip:
    $ pip install -U kaleido
```

**Comment by AlexMayka** [comment]
I downloaded the library data (`kaleido` and `pandas`)

**Comment by Puiching-Memory** [comment]
Has the problem been solved? I installed pandas and kaleido and nothing changed

**Comment by afraniofilho** [comment]
Hi, I had the same problem. I switched my plots to basic matplotlib lib and the build went fine.

**Comment by ndonkoHenri** [comment]
@oelhammouchi please [open](https://github.com/flet-dev/flet/issues/new?template=01_bug_report.yml) a new issue with the details,

**Comment by necromancerWang** [comment]
same problem

**Comment by JonathanTube** [comment]
I get the same problem on Windows 11 desktop enviroment.
